package.xml

```diff
@@ -18,6 +18,7 @@
 
   <build_depend>boost</build_depend>
   <build_depend>pluginlib</build_depend>
+  <build_depend>qtbase5-dev</build_depend>
```

Comment:
The master branch is used to build for all current ROS distros (Indigo - Lunar). But Indigo and Jade are targeting Qt 4. There this can't be added as is to the manifest.